### DISCUSSION TITLE
make installation disk and patching hackmini failed

### DISCUSSION THREAD

**Discussion by kingrhy** [osy86/HaC-Mini#1032] [Q&A] [answered]
macos13 has been released. so today I try to make a install disk use command line method and install hackmini 3.21. but i got failed every time I try to patch install disk. error message as below.
install location selected install u-disk and install options set to default.  someone can tell me why is that and how to fix? thanks
<img width="622" alt="2" src="https://user-images.githubusercontent.com/16399048/197910396-8d090788-55a7-468f-aacf-5cc15ff58e97.png">
 
<img width="622" alt="1" src="https://user-images.githubusercontent.com/16399048/197910392-bd441c50-e1e0-4338-9136-0e64e762c559.png">

**Comment by osy** [comment, marked as answer]
Installer patching no longer worked since Big Sur. Please read the updated guide.

**Reply by kingrhy** [reply]
@osy i am not select 【patching macos installer】 option，but it still failed
<img width="418" alt="image" src="https://user-images.githubusercontent.com/16399048/197966963-12481280-9a43-4fd6-abef-55e71db61ba6.png">

**Reply by osy** [reply]
Post your installer.log in a new issue

**Comment by kingrhy** [comment]
just minutes！let me try one more time！发自我的 iPhone在 2022年10月26日，16:16，osy ***@***.***> 写道：﻿
Post your installer.log in a new issue

**Comment by kingrhy** [comment]
still not work！where i can get install log？发自我的 iPhone在 2022年10月26日，16:16，osy ***@***.***> 写道：﻿
Post your installer.log in a new issue

**Comment by Curls2018** [comment]
I have this problem too, I try to replace multiple disks

**Comment by Curls2018** [comment]
<img width="616" alt="image" src="https://user-images.githubusercontent.com/46666745/198585088-5e5d9eae-bf1c-4de8-84a7-33db24516dc5.png">
<img width="622" alt="image" src="https://user-images.githubusercontent.com/46666745/198585133-d22b2f94-b960-4cee-9218-75b5233378ef.png">

**Comment by Curls2018** [comment]
[install.log](https://github.com/osy/HaC-Mini/files/9888454/install.log)
路径是：/var/log/install.log

**Reply by SpookHCK** [reply]
Have you tried 3.2.2? That fixed that problem for me.

I see this problem:
 install:didFailWithError:Error Domain=PKInstallErrorDomain Code=112 "运行软件包“HaCMini (1).pkg”的脚本时出错。" UserInfo={NSFilePath=./preinstall, NSURL=file:///Users/rongjiexinxi/Desktop/HaCMini%20(1).pkg#HaCMiniCore.pkg, PKInstallPackageIdentifier=com.osy86.hacmini.core, NSLocalizedDescription=运行软件包“HaCMini (1).pkg”的脚本时出错。}
 
when I had this problem, I needed to install Xcode from the App Store 
launch Xcode to finish the install
 
then in terminal:
`xcodebuild -license` accept
`strings` click  install

then run the installer again.
 
 I hope this helps you.

**Comment by kingrhy** [comment]
my alt method，make os12 installation disk install 3.15 version。its work ！ i will wait you guys try until 3.22 on os13 work 发自我的 iPhone在 2022年10月26日，18:29，huaiyong ran ***@***.***> 写道：﻿still not work！where i can get install log？发自我的 iPhone在 2022年10月26日，16:16，osy ***@***.***> 写道：﻿
Post your installer.log in a new issue

**Comment by kingrhy** [comment]
Igot this prompt when i first inatall。 but still failed after xcode installed。 发自我的 iPhone在 2022年10月28日，21:22，SpookHCK ***@***.***> 写道：﻿
Have you tried 3.2.2? That fixed that problem for me.
I see this problem:
install:didFailWithError:Error Domain=PKInstallErrorDomain Code=112 "运行软件包“HaCMini (1).pkg”的脚本时出错。" UserInfo={NSFilePath=./preinstall, NSURL=file:///Users/rongjiexinxi/Desktop/HaCMini%20(1).pkg#HaCMiniCore.pkg, PKInstallPackageIdentifier=com.osy86.hacmini.core, NSLocalizedDescription=运行软件包“HaCMini (1).pkg”的脚本时出错。}
when I had this problem, I needed to install Xcode from the App Store
launch Xcode to finish the install
then in terminal:
xcodebuild -license accept
strings click  install
then run the installer again.
I hope this helps you.

**Reply by SpookHCK** [reply]
Did you open Xcode? and do the terminal steps before trying to install?
Also open a terminal and type `strings` if that just hangs then you have strings installed.. you may need to reboot and try install strings again...
